### ISSUE TITLE
[Bug]: Setting shorts page size to 0 breaks subscription scan

### ISSUE THREAD

**Issue by marvin8** [tubearchivist/tubearchivist#405]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki) and the [readme](https://github.com/tubearchivist/tubearchivist#installing-and-updating), particularly the [common errors](https://github.com/tubearchivist/tubearchivist#common-errors) section.

### Operating System

docker on debian

### Your Bug Report

## Describe the bug
TA fails to re-scan subscriptions. 

## Steps To Reproduce
Set `YouTube Shorts page size:` to "0" to effectively disable shorts.

## Expected behavior
TA successfully scans/re-scans subscriptions and skips any shorts.

### Relevant log output

```shell
[2023-01-15 06:58:27,722: INFO/MainProcess] Task update_subscribed[80b994d7-205d-4091-93c9-d63ae182417f] received                                                                                                                                                                        
[2023-01-15 06:58:28,007: WARNING/ForkPoolWorker-8] UCw3UZn1tcVe7pH3R6C3Gcng: find missing videos.                                                   
[2023-01-15 06:58:30,127: ERROR/ForkPoolWorker-8] Task update_subscribed[80b994d7-205d-4091-93c9-d63ae182417f] raised unexpected: ValueError("'1:False' is not a valid specification")
Traceback (most recent call last):                                                                                                                                                                                                                                                       
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task                                                                                       
    R = retval = fun(*args, **kwargs)                                                                                                                                                                                                                                                    
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__                                              
    return self.run(*args, **kwargs)                                                                                                                 
  File "/app/home/tasks.py", line 61, in update_subscribed                                                                                                                                                                                                                               
    missing_from_channels = channel_handler.find_missing()                                                                                           
  File "/app/home/src/download/subscriptions.py", line 109, in find_missing                                                                          
    last_videos = self.get_last_youtube_videos(channel_id)                                                                                                                                                                                                                               
  File "/app/home/src/download/subscriptions.py", line 56, in get_last_youtube_videos                                                               
    channel = YtWrap(obs, self.config).extract(                                                                                                                                                                                                                                          
  File "/app/home/src/download/yt_dlp_base.py", line 59, in extract                                                                                 
    response = yt_dlp.YoutubeDL(self.obs).extract_info(url)                                                                                          
  File "/root/.local/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1491, in extract_info                                                                                                                                                                                       
    return self.__extract_info(url, self.get_info_extractor(key), download, extra_info, process)                                                     
  File "/root/.local/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1502, in wrapper                                                                                                                                                                                            
    return func(self, *args, **kwargs)                                                                                                               
  File "/root/.local/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1599, in __extract_info                                                 
    return self.process_ie_result(ie_result, download, extra_info)                                                                                                                                                                                                                       
  File "/root/.local/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1728, in process_ie_result                                              
    return self.__process_playlist(ie_result, download)                                                                                              
  File "/root/.local/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1803, in __process_playlist                                            
    entries = resolved_entries = list(entries)                                                                                                                                                                                                                                           
  File "/root/.local/lib/python3.10/site-packages/yt_dlp/utils.py", line 788, in _iter                                                                                                                                                                                                   
    for x in iterable:
  File "/root/.local/lib/python3.10/site-packages/yt_dlp/utils.py", line 3019, in get_requested_items
    for index in self.parse_playlist_items(playlist_items):
  File "/root/.local/lib/python3.10/site-packages/yt_dlp/utils.py", line 3001, in parse_playlist_items
    raise ValueError(f'{segment!r} is not a valid specification')
ValueError: '1:False' is not a valid specification
```


### Anything else?

_No response_

**Comment by bbilly1** [comment]
Thanks for reporting. I've pushed a quick fix in v0.3.2, please update.

**Comment by marvin8** [comment]
Thanks for the quick fix. I have just tested this with the new version 0.3.2 on my end can confirm that scanning subscriptions with shorts disabled now works for me... thank you again.

:heart:
